Jenkinsfile

```diff
@@ -1,8 +1,15 @@
-def preBuildStages(String stageName) {
+def global = [:]
```

Comment:
What's this?

Reply:
declaration of a global object. Needed to pass the versionFile object between stages run by different agents/nodes